src/bidiMapper/modules/browser/BrowserProcessor.ts

```diff
@@ -100,4 +106,34 @@ export class BrowserProcessor {
       ],
     };
   }
+
+  async #getWindowInfo(targetId: string): Promise<Browser.ClientWindowInfo> {
+    const windowInfo = await this.#browserCdpClient.sendCommand(
+      'Browser.getWindowForTarget',
+      {targetId},
+    );
+    return {
+      // Is not supported in CDP yet.
+      active: false,
+      clientWindow: `${windowInfo.windowId}`,
```

Comment:
I think the spec returns each window only once https://w3c.github.io/webdriver-bidi/#command-browser-getClientWindows